doc/specs/#140 - ZIP Support.md

```diff
@@ -0,0 +1,135 @@
+---
+author: Alexis Bekhdadi @midoriiro
+created on: 2021-11-16
+last updated: 2021-11-16
+issue id: 140
+---
+
+# Spec Title
+
+For [#140](https://github.com/microsoft/winget-cli/issues/140)
+
+## Abstract
+
+This feature adds to winget-cli the ability to install application delivered through a ZIP archive.
+
+## Inspiration
+
+A bunch of applications are only provided by developer's through a ZIP archive. There are three cases:
+1) The archive contain only one binary that need do to be extracted and add to PATH env var
+2) The archive contain an installer that need to be executed and let do his job
+3) The archive contain a complex hierarchy that need do to be extracted and add specific folder to PATH env var
```

Comment:
```suggestion
3) The archive contains more than one file,
```

Reply:
The cases I see:
1. Archive is the set of files that should be placed on the system (both your 1 and 3)
2. Archive is a container for the installer, and is itself temporary (your 2)
3. [New] Archive is basically both of the above; the files themselves need to be kept, but there is some registration step that is needed.